tests/cases/ref-list.xml

```diff
@@ -4,8 +4,14 @@
 <?expected-error message="<ref-list> is ignored." node="/ref-list"?>
 <?expected-error message="<title> in <ref-list> is ignored." node="//title"?>
 <?expected-error message="<p> in <ref-list> is ignored." node="//p"?>
+<?expected-error message="<fig> in <ref-list> is ignored." node="//fig"?>
```

Comment:
Looks like a nightmare to update `is ignored` errors every time an element is added?

Reply:
It's a bit of a pain, but feel it's worth the effort (this library benefits from precision).